spec/controllers/media_objects_controller_spec.rb

```diff
@@ -782,6 +785,26 @@
       media_object.reload
       expect(media_object.descMetadata.media_type).to eq(["video/mp4"])
     end
+
+    context 'large objects' do
+      let!(:media_object) do
+        mo = FactoryGirl.create(:published_media_object)
+        10.times { FactoryGirl.create(:master_file_with_derivative, mediaobject: mo) }
```

Comment:
When I ran this test on my laptop, I did it with 50 MasterFiles. But this one test took more than 10 minutes to run because of all the instrumentation rspec injects, so I reduced it to 10, which is enough to get it to fail _sometimes_ under the old version of the code.